tools/docker/jetpack-docker-config-default.yml

```diff
@@ -0,0 +1,58 @@
+# Default configuration. May be overridden for specific environments by sections below.
+default:
+  # Volumes to mount inside the environment. Keys are the local paths, which may be absolute
+  # or relative to the monorepo root. Values are the paths inside the Docker environment, and
+  # must be absolute.
+  volumeMappings:
+    tools/docker/wordpress: /var/www/html
+    .: /usr/local/src/jetpack-monorepo
+    tools/docker/mu-plugins: /var/www/html/wp-content/mu-plugins
+
+  # Extra configuration (none by default). Anything set under this key will be written out as
+  # an extra docker-compose configuration file.
+  extras:
```

Comment:
If we're copying tools/docker/docker-compose.dev.yml into this file, we could copy tools/docker/docker-compose.yml into here too.

But maybe that's not a good idea, it might just make things more confusing.

Reply:
I don't have a strong opinion on this one, but I think having a "default" static compose file probably makes it a bit easier to understand what's going on under the hood.